website/docs/r/google_kms_crypto_key_iam_binding.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
You have to update also the `website/google.erb` file to add an entry in the left menu panel

Reply:
Added. There are now enough KMS resources in there that it might be a good idea to pull out another submenu - what do you think? 

Also, how exactly do I build these docs? I'm basically slamming changes in there and hoping it works at this point ...

Reply:
I am planning on reorganizing the docs around IAM next week. I want to discuss it with @paddycarver and @danawillow  first. I would say for now it is fine the way you did it.

About building the docs, @paddycarver do we have documentation somewhere about how to easily build the docs?

Reply:
@tragiclifestories The documentation on how to build the website is there:
https://github.com/hashicorp/terraform-website